### PR TITLE
NOT NEEDED ANY LONGER FOR WEB WORKERS Eject from react-scripts

### PR DESCRIPTION
**Pull request by leslielazzarino**
# Probably we just want to avoid all the hassle of this change
and just don't do it, as this PR would only be a start and there isn't a strong reason for it any longer.

Signed-off-by: Leslie Lazzarino <leslie.lazzarino@tngtech.com>

### Summary of changes

Eject from react-scripts

### Context and reason for change

Required to use web workers

### How can the changes be tested

Check that all yarn scripts still work

### CODE REVIEW

File: config/env.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// SPDX-FileCopyrightText: Facebook, Inc. and its affiliates
```
Comment (benedikt-richter) [inline config/env.js:1]: I'm not sure we have the copyright for these files

File: config/env.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+// SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
+//
+// SPDX-License-Identifier: Apache-2.0
```
Comment (benedikt-richter) [inline config/env.js:4]: Not sure this license applies here

File: config/env.js (hunk 1 above)
Comment (leslielazzarino) [reply config/env.js:1]: I think we need a different header, with both create-react-app compl. info and ours..

File: config/env.js (hunk 2 above)
Comment (nicarl) [reply config/env.js:4]: We should be careful with such files generated by the eject command. @maxhbr can you give us guidance on this?

File: package.json [hunk 3]
```diff
@@ -32,10 +32,10 @@
     "stream-json": "^1.7.2",
     "upath": "^2.0.1",
     "uuid": "^8.3.0",
-    "write-file-atomic": "^3.0.3",
-    "react-scrollbar-size": "4.0.0"
+    "write-file-atomic": "^3.0.3"
```
Comment (nicarl) [inline package.json:35]: I think you need to rebase to latest master

File: src/react-app-env.d.ts [hunk 4]
```diff
@@ -3,4 +3,77 @@
...
+}
+
+declare module '*.module.scss' {
+  const classes: { readonly [key: string]: string };
+  export default classes;
+}
+
+declare module '*.module.sass' {
+  const classes: { readonly [key: string]: string };
+  export default classes;
```
Comment (benedikt-richter) [inline src/react-app-env.d.ts:78]: this is not needed, I guess

File: src/react-app-env.d.ts [hunk 5]
```diff
@@ -3,4 +3,77 @@
...
+
+declare module '*.avif' {
+  const src: string;
+  export default src;
+}
+
+declare module '*.bmp' {
+  const src: string;
+  export default src;
+}
```
Comment (benedikt-richter) [inline src/react-app-env.d.ts:28]: this is not needed, I guess

File: jest.config.js [hunk 6]
```diff
@@ -2,3 +2,49 @@
...
+  setupFiles: ['react-app-polyfill/jsdom'],
+  setupFilesAfterEnv: [],
+  testMatch: [
+    '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
+    '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
+  ],
+  testEnvironment: 'jsdom',
+  transform: {
+    '^.+\\.(js|jsx|mjs|cjs|ts|tsx)$': '<rootDir>/config/jest/babelTransform.js',
+    '^.+\\.css$': '<rootDir>/config/jest/cssTransform.js',
```
Comment (benedikt-richter) [inline jest.config.js:18]: this is not needed, I guess

File: config/jest/cssTransform.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: Facebook, Inc. and its affiliates
```
Comment (benedikt-richter) [inline config/jest/cssTransform.js:1]: this file is not needed, I guess

File: LICENSES/MIT.txt [hunk 8]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (benedikt-richter) [inline LICENSES/MIT.txt:1]: why do we add this one?

**Comment by benedikt-richter** [comment]
Some of the following dependencies might be unused:

Unused dependencies
* @mui/lab
Unused devDependencies
* babel-plugin-named-asset-import
* browserslist
* identity-obj-proxy
* jest-resolve
* postcss-flexbugs-fixes
* postcss-normalize
* postcss-preset-env
* prompts
* sass-loader
* tailwindcss
* wait-on

**Pull request closed without merging**